adding_species_manuscript.tex

```diff
@@ -367,6 +367,9 @@ \section*{Introduction}\label{introduction}}
 generally reduce variance relative to independent simulations of equivalent genetic material.
 This has a particularly striking effect in long stretches of low recombination rates,
 as observed for instance on the long arm of human chromosome 22 \citep{Dawson2002}.
+In bacteria, a similar effect occurs due to genome-wide linkage that is broken only
+by gene conversion of short segments {[TODO: citation?]}.
+
```

Comment:
This might be a suitable citation:
[10.1016/j.tim.2010.04.002](https://doi.org/10.1016%2Fj.tim.2010.04.002)

Reply:
@fbaumdicker , this paragraph has been extensively edited to better capture the new way that non-crossover recombination is modeled in stdpopsim. See the latest PR: https://github.com/popsim-consortium/adding-species-manuscript/pull/68. Let me know if you have any comments on current version. BTW, I'm using this reference now, in a similar context.

Reply:
Ah yes, thanks for the link to the newest PR. I have added one small comment to the latest PR. Besides that suggestion I think @petrelharp has already addressed all other points.